projects/angular/src/accordion/_properties.accordion.scss

```diff
@@ -4,80 +4,49 @@
 
-      --clr-accordion-border-color: var(--clr-color-neutral-400);
-      --clr-accordion-border-radius: var(--clr-global-borderradius);
+      --clr-accordion-border-left-color-complete: #{tokens.$cds-alias-status-success};
+      --clr-accordion-border-left-color-error: #{tokens.$cds-alias-status-danger};
 
-      --clr-accordion-border-left-color: #{accordion-variables.$clr-accordion-border-left-color};
-      --clr-accordion-border-left-width: #{variables.$clr_baselineRem_0_25};
-      --clr-accordion-border-left-color-current: var(--clr-accordion-border-left-color);
-      --clr-accordion-border-left-color-complete: var(--clr-accordion-complete-color);
-      --clr-accordion-border-left-color-error: var(--clr-accordion-error-color);
+      --clr-accordion-header-left-indicator: inset #{tokens.$cds-global-space-5} 0 0 #{tokens.$cds-alias-status-neutral-tint};
+      --clr-accordion-header-left-current-indicator: inset #{tokens.$cds-global-space-5} 0 0 #{tokens.$cds-alias-object-interaction-background-highlight};
 
-      // compound properties
-      --clr-accordion-header-left-indicator: inset
-        var(--clr-accordion-border-left-width)
-        0
-        0
-        var(--clr-accordion-border-left-color);
-      --clr-accordion-header-left-current-indicator: inset
-        var(--clr-accordion-border-left-width)
-        0
-        0
-        var(--clr-accordion-border-left-color-current);
-      --clr-accordion-header-left-complete-indicator: inset
-        var(--clr-accordion-border-left-width)
-        0
-        0
-        var(--clr-accordion-border-left-color-complete);
-      --clr-accordion-header-left-error-indicator: inset
-        var(--clr-accordion-border-left-width)
-        0
-        0
-        var(--clr-accordion-border-left-color-error);
+      --clr-accordion-header-gap: #{tokens.$cds-global-space-5};
 
-      --clr-accordion-header-gap: 0;
-      --clr-accordion-status-icon-width: #{variables.$clr_baselineRem_0_8};
+      --clr-accordion-step-title-min-width: 0;
 
-      --clr-accordion-content-color: #{accordion-variables.$clr-accordion-content-color};
-      --clr-accordion-content-font-size: #{accordion-variables.$clr-accordion-content-font-size};
-      --clr-accordion-description-margin-left: #{accordion-variables.$clr-accordion-description-margin-left};
-      --clr-accordion-step-title-min-width: #{accordion-variables.$clr-accordion-step-title-min-width};
-      --clr-accordion-step-title-font-weight: #{accordion-variables.$clr-accordion-step-title-font-weight};
-      --clr-accordion-step-status-margin-left: #{accordion-variables.$clr-accordion-step-status-margin-left};
-      --clr-accordion-step-title-margin-left: #{accordion-variables.$clr-accordion-step-title-margin-left};
+      --clr-accordion-step-status-margin-left: #{tokens.$cds-global-space-7};
+      --clr-accordion-step-title-margin-left: #{tokens.$cds-global-space-9};
```

Comment:
We can just use the core values inside _accordion. Exposed properties are not needed for these